### PR TITLE
Tickets/dm36343: Updates for the integrationtests package

### PR DESCRIPTION
**Pull request by rbovill**

### CODE REVIEW

File: cycle/cycle.env
Comment (mareuter) [inline cycle/cycle.env:105]: This should be moved to `develop` since that branch has been delete otherwise this won't work anymore.

**Review by mareuter — APPROVED** [review]
One comment to address, but I think this looks fine content wise. I'd like either Tiago or Wouter to make sure the commits comply with current expectations. So please don't merge until one of them approves.

File: build/integrationtests/Dockerfile
Comment (tribeiro) [inline build/integrationtests/Dockerfile:15]: are you sure `PyYAML` is not available in conda? I know it is small detail but it is better to try installing everything we can from conda.

File: build/integrationtests/Dockerfile
Comment (tribeiro) [inline build/integrationtests/Dockerfile:23]: Any chance `robotframework_EFD` might become a conda package one day? Then you don't have to be messing with `PYTHONPATH`.

File: cycle/cycle.env
Comment (tribeiro) [inline cycle/cycle.env:105]: yeah, so we usually want to have only tags/release numbers in cycle.env... any chance you can tag what you want to deploy instead of using develop? note, for instance that having develop branch here and trying to rebuild the container would not generate new artifacts unless you clean the container.. it also doesn't tell us which version of the package you are running.

File: build/integrationtests/Dockerfile
Comment (tribeiro) [inline build/integrationtests/Dockerfile:10]: Consider moving this to line 20. Basically if you leave it here and you change the input docker will rerun the following commands in line 12 and 17 unnecessary recreating layers.

**Review by tribeiro — COMMENTED** [review]
Here some general comment.. some of them are "future requests" for you to keep in mind, like considering make  a conda package for `robotframework_EFD`. 

There is one thing that I really want to enforce though, which is avoid using branch names in cycle.env. Please, consider using a version tag.

**Review by tribeiro — APPROVED** [review]
Looks good! thanks for the updates!

**Pull request merged**
